metrics/prometheus/prometheus.yml

```diff
@@ -0,0 +1,19 @@
+global:
+  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+scrape_configs:
+  # - job_name: 'prometheus'
+  #   static_configs:
+  #   - targets: ['localhost:9090']
```

Comment:
leftovers?

Reply:
Yeap, that is part of the default config file. Thanks!